src/docs/background/off-session-payments/index.md

```diff
@@ -72,3 +96,56 @@ provider to post a review of the customer. In addition, you may consider
 disallowing customers to remove their stored payment card in your UI
 implementation, if they have ongoing transactions for which they have
 not yet been charged.
+
+## Considerations about implementation in FTW templates
+
+If you want to implement
+[the example process](https://github.com/sharetribe/flex-example-processes)
+in your user interface, there are multiple ways to do so. If your user
+interface is based on one of the FTW templates, here are a few things
+worth considering.
+
+### Transitions and states
+
+[Transitions and states](/tutorial-transaction-process/create-transaction-process/#update-client-app)
+are used in the template as conditions for several behaviors, including
+redirects and displayed content. The
+[transaction resource](https://www.sharetribe.com/api-reference/marketplace.html#transaction-resource-format)
+contains information about the transaction's last transition and its
+timestamp.
+
+### Separating order from payment
+
+In the default transaction process and default FTW flow, the order is
+initiated and processed on
+[CheckoutPage.js](https://github.com/sharetribe/ftw-daily/blob/6e856692d0b7f3a7a1110894967a229a70ae0ce0/src/containers/CheckoutPage/CheckoutPage.js#L355)
+using
+[initial values from ListingPage.js](https://github.com/sharetribe/ftw-daily/blob/6e856692d0b7f3a7a1110894967a229a70ae0ce0/src/containers/ListingPage/ListingPage.js#L122).
+Since the off-session payment process separates initiating the order
+(i.e. creating a booking, setting line items in a privileged transition)
+from payment (creating and further processing Stripe payment intent), it
+is important to pay attention to the way you want to handle that
+separation.
+
+- What happens when user clicks 'Request to book' on ListingPage.js?
+- Where is the API call made to create a booking and set line items?
+
+### Handling delayed manual payment
+
+If the automatic payment succeeds, the customer does not need to take
+further action on the transaction before the review process. Manual
+payment, on the other hand, does require a new user flow in the FTW
+template. CheckoutPage.js is set up to
+[handle payments toward Stripe](https://github.com/sharetribe/ftw-daily/blob/6e856692d0b7f3a7a1110894967a229a70ae0ce0/src/containers/CheckoutPage/CheckoutPage.js#L795),
+so the simplest option is that once the customer has confirmed the
+booking and triggered the transition to create a payment intent, they
```

Comment:
Should this say that the provider has confirmed the booking? Could it be mentioned here something like _once a payment has not automatically gone through and the customer has manually triggered the transition to create a payment intent..._ or something like that? Would that make sense here?

Reply:
The original thought was 'once the customer has confirmed the _payment_', but the more explicit wording was better, so fixed in that style!